Mathlib/AlgebraicTopology/SimplicialSet/Degenerate.lean

```diff
@@ -232,4 +232,121 @@ lemma unique_nonDegenerate_map (x : X _⦋n⦌) {m : ℕ}
 
 end
 
+namespace Subcomplex
+
+variable {X} (A : X.Subcomplex)
+
+lemma mem_degenerate_iff {n : ℕ} (x : A.obj (op (.mk n))) :
+    x ∈ degenerate A n ↔ x.1 ∈ X.degenerate n := by
+  rw [SSet.mem_degenerate_iff, SSet.mem_degenerate_iff]
+  constructor
+  · rintro ⟨m, hm, f, _, ⟨y, rfl⟩⟩
```

Comment:
```suggestion
  · rintro ⟨m, hm, f, _, y, rfl⟩
```
Recall that `⟨u₁, u₂, …, uₙ⟩` is secretely `⟨u₁, ⟨u₂, …, ⟨uₙ₋₁,  uₙ⟩⟩⟩`. If you want, there are many more instances below. Your call if you think the original is more readable.